apps/blog/src/api/mappers/post.ts

```
@@ -0,0 +1,56 @@
+    category: DbPostCategory;
+  }): PostCategoryEnum {
+    switch (category) {
+      case "HEALTH":
+        return PostCategoryEnum.HEALTH;
+      case "PHILOSOPHY":
+        return PostCategoryEnum.PHILOSOPHY;
+      case "SOFTWARE":
+        return PostCategoryEnum.SOFTWARE;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)